lib/manager/batect/extract.ts

```diff
@@ -103,3 +136,39 @@ export function extractPackageFile(
     return null;
   }
 }
+
+export async function extractAllPackageFiles(
+  config: ExtractConfig,
+  packageFiles: string[]
+): Promise<PackageFile[] | null> {
+  const filesToExamine = [...packageFiles];
+  const filesAlreadyExamined = [];
```

Comment:
```suggestion
  const filesAlreadyExamined: string[] = [];
```
Better use a `Set<string>` here